index.html

```diff
@@ -298,6 +300,9 @@ <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
         <li> A <a>PerformanceEntryList</a> object called the <dfn>observer
         buffer</dfn> that is initially empty.
         </li>
+        <li>A sequence of strings called <dfn>supported entry types</dfn> that is
```

Comment:
s/supported entry types/observed entry types/? 

I think we should reserve `supported entry types` for the global "which entry types does performance timeline support"?

Reply:
Done